processor/src/main/java/org/mapstruct/ap/internal/model/PropertyMapping.java

```diff
@@ -468,12 +474,13 @@ private Assignment assignToPlainViaAdder( Assignment rightHandSide) {
 
             Assignment result = rightHandSide;
 
+            String adderWrapperName = sourcePropertyName == null ? targetPropertyName : sourcePropertyName;
```

Comment:
`adderIteratorName` ?

Reply:
Thank you very much. I had troubles coming up with a nice name, your suggestion is perfect